drivers/crypto/crypto_ataes132a.c

```diff
@@ -779,7 +779,8 @@ static int do_block(struct cipher_ctx *ctx, struct cipher_pkt *pkt)
 	return ataes132a_aes_ecb_block(dev, key_id, pkt);
 }
 
-int ataes132a_session_free(struct device *dev, struct cipher_ctx *session)
+static int ataes132a_session_free(struct device *dev,
+				struct cipher_ctx *session)
```

Comment:
align the parameter to the last '(' + 1 space.

Apply that everywhere.

Reply:
ok got it.